### DISCUSSION TITLE
Phalcon 5.0.0RC1 autocomplete for PHPStorm

### DISCUSSION THREAD

**Discussion by QUAD69** [phalcon/phalcon#254] [Q&A] [answered]
How can I enable code highlighting/autocomplete in phpstorm for phalcon v5? The marketplace has an extension for version 4 only(

**Comment by Jeckerson** [comment, marked as answer]
There is separate composer package for that - https://github.com/phalcon/ide-stubs/releases

```
composer require --dev phalcon/ide-stubs:v5.0.0-RC1
```
